### PR TITLE
Triplelift: Expand Video Formats

### PR DESCRIPTION
**Pull request by SyntaxNode**
Created on behalf of Triplelift.

### CODE REVIEW

**Comment by nllerandi3lift** [comment]
Thanks looks good.

File: adapters/triplelift/triplelift.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type TripleliftRespExt struct {
 
 func getBidType(ext TripleliftRespExt) openrtb_ext.BidType {
 	t := ext.Triplelift.Format
-	if t == 11 {
+	if t == 11 || t == 12 || t == 17 {
```
Comment (VeronikaSolovei9) [inline adapters/triplelift/triplelift.go:29]: Would it be better to extract these magic numbers to constants?

**Review by bsardo — APPROVED** [review]
LGTM

File: adapters/triplelift/triplelift.go (hunk 1 above)
Comment (SyntaxNode) [reply adapters/triplelift/triplelift.go:29]: Ideally, I agree. I do not know what these format values represent. I don't think it matters too much in this context and I'm personally ok with magic numbers in this instance. Do you feel strongly about this suggestion or is this ok?

Comment (nllerandi3lift) [reply adapters/triplelift/triplelift.go:29]: These are just different ids indicating instream, outstream, and one other video format. I don't think it's super relevant to have that indicated here.

**Review by VeronikaSolovei9 — APPROVED** [review]

**Comment by nllerandi3lift** [comment]
@bretg this resolved a rendering issue we were seeing when testing Outstream (recently added format). Wondering if you can help port to Java.

**Comment by bretg** [comment]
Adding @net-burst for FYI. Oleksandr, please consider this one of the higher priority porting tasks. Thanks.

**Comment by Net-burst** [comment]
Done in https://github.com/prebid/prebid-server-java/pull/1996

**Pull request merged**